group_vars/all/globals.yml

```diff
@@ -34,3 +34,5 @@ wal_e_pgdata_dir: "/var/lib/postgresql/{{ postgresql_version }}/main/"
 postgresql_archive_command: >
   /usr/bin/envdir {{ wal_e_envdir }} /usr/local/bin/wal-e
   {% if wal_e_aws_instance_profile %}--aws-instance-profile{% endif %} wal-push %p
+
+postgres_host: "{{ deploy_env }}-postgres-master.{{ domain_name }}"
```

Comment:
Yes, I agree. These names might be confusing. 

Currently:
 * `postgres_host`: DNS alias of the postgres master node
 * `postgres_master`: IP of the postgres master node.

I suggest something like:

 * `postgres_master_host`: DNS alias of the postgres master node
 * `postgres_master_ip`: IP of the postgres master node.